kitsune/wiki/jinja2/wiki/includes/sidebar_modules.html

```diff
@@ -64,9 +64,16 @@ <h3 class="sidebar-subheading large-only">{{ _('Editing Tools') }}</h3>
             </li>
         </ul>
       </li>
+      {% else %}
+      <span class="details-heading"></span>
+      <ul class="sidebar-nav--list">
+        <li>
+          {{ show_for((document or parent).get_products()) }}
```

Comment:
Shouldn't this be under `{% if include_showfor %}`?

Reply:
On second thought instead of adding an else statement, can't we just move the `showfor` section outside of the `is_authenticated` check?

Reply:
Ah, good point, I'd overlooked the `include_showfor` condition.